.gitmodules

```diff
@@ -354,6 +354,9 @@
 	path = toSketch
 	url = https://github.com/KeithSloan/toSketch.git
 	branch = main
+[submodule "taack-plm-freecad"]
+	path = taack-plm-freecad
+	url = https://github.com/Taack/taack-plm-freecad.git
```

Comment:
```suggestion
[submodule "taack-plm-freecad"]
	path = taack-plm-freecad
	url = https://github.com/Taack/taack-plm-freecad.git
	branch = main
```

Reply:
Don't forget the branch line (@luzpaz IMO we should add a `branch = ` line to all of them, so it's obvious that you should have it. It defaults to `master` for historical reasons, but most new devs use `main`)